package.json

```diff
@@ -21,8 +21,11 @@
   },
   "homepage": "https://github.com/pink-bunny/tmdb#readme",
   "dependencies": {
+    "@babel/plugin-transform-async-to-generator": "^7.8.3",
```

Comment:
Babel plugins should be in devDependencies as it used only for create js bundle in dev environment

Reply:
Fixed